schema/private.openapi.yaml

```diff
@@ -356,6 +356,18 @@ components:
       type: string
       minLength: 1
 
+    HostIdWithAnsibleHost:
```

Comment:
Can this be renamed to something like `HostDetails` or something a bit more generic?

Its possible that we will need to add more info about the hosts later on and then this name makes less sense.

Naming is hard  :(

Reply:
I changed this to `HostAtribute`. Unfortunately, `HostDetails` is used quite extensively thoughout the codebase to mean a different thing, so didn't want to add to the confusion here.